Containers/mastercontainer/Caddyfile

```diff
@@ -6,17 +6,29 @@
     storage file_system {
         root /mnt/docker-aio-config/caddy/
     }
+
+    log {
+        level ERROR
+    }
 }
 
 http://:80 {
   redir https://{host}{uri}
 }
 
-https://:8443 {
+# Match only host names and not ip-addresses:
+https://*.*:8443,
+https://*.*.*:8443,
+https://*.*.*.*:8443,
+https://*.*.*.*.*:8443,
+https://*.*.*.*.*.*:8443 {
```

Comment:
@Zoey2936 this would match at max 1.2.3.4.yourdomain.com:8443. Do you think this is fine or should we add another level?

Reply:
This should be enough, the least people will use more than a subdomain of a subdomain

Reply:
Okay good. Thanks!